cfme/tests/configure/test_default_views_infra.py

```diff
@@ -50,6 +50,20 @@ def set_and_test_default_view(group_name, view, page):
     assert tb.is_active(view), "{} view setting failed".format(view)
     DefaultView.set_default_view(group_name, old_default)
 
+
+def check_vm_visibility():
+    view = navigate_to(Vm, 'Explore')
+    value = view.sidebar.vmstemplates.tree.read_contents()
```

Comment:
I suppose there should be better methods to find necessary item in menu change reading whole contents and operating with indexes of multilevel list

Reply:
As the name and other thing changes with respect to the providers, the only thing that didn't change was the index of the list, so I though of implementing it this way.